src/System.Buffers.Primitives/System/Buffers/OwnedBuffer.cs

```diff
@@ -69,7 +40,11 @@ public void Dispose()
         public abstract void Retain();
 
         public abstract void Release();
-
         #endregion
+
+        protected static unsafe void* Add(void* pointer, int offset)
```

Comment:
Sounds good to me. S.R.CS.Unsafe has both `void*` and `ref T` overloads for most methods. `void*` overloads for Add and Subtract are missing for no good reason.